### PR TITLE
upgrade rocket library to 0.5 version.

### PR DESCRIPTION
**Pull request by xiaoxigua-1**
upgrade !!

### CODE REVIEW

**Comment by SiongSng** [comment]
thanks

File: Cargo.toml [hunk 1]
```diff
@@ -8,8 +8,7 @@ version = "0.1.0"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-rocket = "0.4.10"
-rocket_contrib = "0.4.10"
+rocket = { version = "0.5.0-rc.1", features = ["json"] }
```
Comment (SiongSng) [inline Cargo.toml:11]: Please upgrade to 0.5.0-rc.2

**Review by SiongSng — CHANGES_REQUESTED** [review]

File: Cargo.toml (hunk 1 above)
Comment (xiaoxigua-1) [reply Cargo.toml:11]: no 0.5.0-rc.2
https://crates.io/crates/rocket/versions

Comment (SiongSng) [reply Cargo.toml:11]: sorry

**Review by SiongSng — APPROVED** [review]
LGTM!

**Pull request merged**